docs/.vuepress/components/cdr-doc-example-code-pair.vue

```diff
@@ -0,0 +1,292 @@
+            return codeNode !== false;
+          });
+        }
+
+        return codeNode
+      },
+      renderExampleFromTemplate (template, mountId) {
+        // This method calls the Vue compiler directly to render the VUE template content as HTML
+        var tempComponent = new Vue({
+          template: template,
+          parent: this,
+        }).$mount(mountId)
+      }
+    }
+  }
+</script>
+<style lang="scss">
+  @import '../theme/styles/cdr-tokens.scss';
+  @import '../theme/styles/cdr-doc-tokens.scss';
+  $cdr-doc-code-snippet-actions-background-color: $ice-age;
+
+  .cdr-doc-example-code-pair {
+    border: $cdr-doc-border-separator;
+    border-radius: $cdr-doc-border-radius-default;
+    margin-bottom: $space-1-x;
+
+    // code snippet style overrides for composite component
+    .cdr-doc-code-snippet {
+      border: 0;
+      margin-bottom: 0;
+    }
+
+    .cdr-doc-code-snippet__actions {
+      border: 0;
+      border-radius: 0;
+    }
+
+    div[class^='language-'] {
+      border-bottom: 0;
+      border-left: 0;
+      border-right: 0;
+    }
+  }
+
+  .cdr-doc-example-code-pair__item {
+    align-items: center;
+    border-bottom: $cdr-doc-border-separator;
+    display: flex;
+    justify-content: center;
+    padding: $inset-2-x;
+    position: relative;
+
+    &:last-child {
+      border-bottom: 0;
+    }
+  }
+
+  .cdr-doc-example-code-pair--not-interactive .cdr-doc-example-code-pair__item-example {
+    pointer-events: none; // prevent interaction
+  }
+
+  .cdr-doc-example-code-pair__item-background--dark {
+    background: $lost-in-space;
+  }
+
+  .cdr-doc-example-code-pair__code {
+    display: none;
+  }
+
+  .cdr-doc-example-code-pair__item-label {
+    @include redwood-display-10;
+    bottom: $space-half-x;
+    position: absolute;
+    right: $space-half-x;
+
+    .cdr-doc-example-code-pair__item-background--dark & {
+      color: $clean-slate;
+    }
+  }
+
+  .cdr-doc-example-code-pair__item-background-toggle {
+    display: flex;
+    position: absolute;
+    right: $space-half-x;
+    top: $space-half-x;
+  }
+
+  .cdr-doc-item-background-toggle__button {
+    @include redwood-display-20;
+    background-color: $clean-slate;
+    border: solid 1px $holy-smoke;
+    border-right-width: 0;
+    cursor: pointer;
+    padding: $space-half-x;
+    padding-bottom: $space-half-x-squish;
+    padding-top: $space-half-x-squish;
+
+    &:first-child {
+      border-radius: $cdr-doc-border-radius-default 0 0 $cdr-doc-border-radius-default;
+    }
+    
+    &:last-child {
+      border-radius: 0 $cdr-doc-border-radius-default $cdr-doc-border-radius-default 0;
+      border-right-width: 1px;
+    }
+  }
+  
+  .cdr-doc-item-background-toggle__button--active {
+    background: $partly-cloudy;
+  }
+
+  .cdr-doc-item-background-toggle__input {
+    position: absolute !important;
+    height: 1px; width: 1px; 
+    overflow: hidden;
+    clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
```

Comment:
Not sure this is needed, these browsers have long been removed from the support matrix maintained by test at REI -

Reply:
I copy/pasted that code snippet from a google search on "accessibly hidden" text. I think those comments are stating that this is supported all the way back to IE6 and IE7, not that the `clip` property is used _to_ support IE6 and IE7. I'll remove the comment to reduce confusion.

Reply:
Nope, I was wrong about this. I see that this was an old, alternate syntax. I've removed this line.